gamestonk_terminal/main_helper.py

```diff
@@ -52,6 +52,8 @@ def print_help(s_ticker, s_start, s_interval, b_is_market_open):
     print(
         "   sen         sentiment of the market, \t from: reddit, stocktwits, twitter"
     )
+    print("   fred        Data from Federal Reserve Bank of St.Louis")
```

Comment:
I'd try to keep the structure of the main menu

<img width="598" alt="Captura de ecrã 2021-03-16, às 20 13 53" src="https://user-images.githubusercontent.com/25267873/111376325-37d7b800-8697-11eb-8e5c-61ca33cca3c5.png">